metadata_layer.py

```diff
@@ -1587,6 +1598,8 @@ def _key(classification):
                     original_resource=original_resource,
                     transformation_settings=link.transformation_settings,
                 )
+                work_requires_new_presentation_edition = True
```

Comment:
This is the exception. Previously, if you added links, we didn't set made_core_changes. But if you add links there's the possibility that the work's cover image has changed, so the work does require a new presentation edition.

I'm don't understand 100% how we used made_core_changes outside this method, but it looks like it was sometimes used to trigger an immediate presentation edition recalculation, so it _should_ have been set to true when new links were added.

Reply:
Isn't the summary also a link? So this could require a full recalculation.